README.md

```diff
@@ -1,11 +1,23 @@
 # InfluxDB [![CircleCI](https://circleci.com/gh/influxdata/influxdb.svg?style=svg)](https://circleci.com/gh/influxdata/influxdb)
 
-This is the for InfluxDB 2.0 OSS, which includes parts that used to be known as Chronograf and Kapacitor.
+InfluxDB is an open source time series platform. This includes APIs for storing and querying data, processing it in the background for ETL or monitoring and alerting purposes, user dashboards, and visualizing and exploring the data and more. The master branch on this repo now represents InfluxDB 2.0, which includes functionality for Kapacitor (background processing) and Chronograf (the UI). If you are looking for the 1.x line of releases, there are branches for each of those. InfluxDB 1.8 will be the next (and likely last) release in the 1.x line and the [working branch is here](https://github.com/influxdata/influxdb/tree/1.8).
 
-If you are looking for the [InfluxDB 1.x Go Client, we've created a new repo](https://github.com/influxdata/influxdb1-client) for that.
+If you are looking for the [InfluxDB 1.x Go Client, we've created a new repo](https://github.com/influxdata/influxdb1-client) for that. There will be a Go client for the 2.0 API coming very soon.
+
+## State Of the Project
+
+InfluxDB 1.7.3 is the current stable release and recommended for production use. InfluxDB 2.0 (what's in the master branch) is currently in the alpha stage. This means that it is not recommended for production usage. There may be breaking API changes, breaking changes in the [Flux language](https://github.com/influxdata/flux), changes in the underlying storage format that will require you to wipe out all your old data, and significant changes to the UI. We will be cutting weekly versioned releases every week starting in the first week of February. There will also be nightly builds.
+
+Once we close on the final feature set of what will be in the first release of InfluxDB in the 2.x line, we will move into the beta phase. At that point we won't be making breaking changes to the API or the Flux language. However, it will still not be recommended for production usage. During the beta period we will focus on bug fixes, performance, and additive features (where time permits).
```

Comment:
Replace: At that point we won't be making breaking changes to the API or the Flux language. 

With: At that point, we don't plan on making breaking changes to the API or the Flux language.  However, it may be necessary to do so. We will do our best to minimize this and clearly communicate ANY and ALL changes in this regard via the change log.